edl/ccf.edl

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache 2.0 License.
 
 enclave {
+    from "openenclave/edl/sgx/platform.edl" import *;
```

Comment:
New, see: https://github.com/openenclave/openenclave/blob/master/docs/SystemEdls.md

Reply:
Nice. What's the use for this in our case?

Reply:
We need it to be able to link, since we use several of these ocalls!